.circleci/config.yml

```diff
@@ -1573,7 +1573,7 @@ jobs:
 
   test-e2e-react:
     docker:
-      - image: mcr.microsoft.com/playwright:v1.33.0-focal
+      - image: mcr.microsoft.com/playwright:v1.38.0-jammy
```

Comment:
if you update playwright container, could you update playwright version? the version number should be the same